lib/evss/pciu/phone_number_response.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'evss/response'
+
+module EVSS
+  module PCIU
+    class PhoneNumberResponse < EVSS::Response
+      attribute :phone, Hash
```

Comment:
For Letters we did all the data munging in the response rather than the serializer, it's neither here nor there but if you're already doing some hash digging in the constructor maybe all the data transformations could go here, then the serializer is just a list of attributes.